lib/parsers/index.ts

```diff
@@ -206,29 +217,38 @@ async function getDependenciesFromReferenceInclude(manifestFile, includeDev: boo
       depInfo[propertyValuePair[0]] = propertyValuePair[1];
     }
 
-    const dependency: PkgTree = {
+    const dependency: DotnetDepsPkgTree = {
       // TODO: correctly identify what makes the dep be dev only
       depType: DepType.prod,
       dependencies: {},
       name: depName,
       version: depInfo.Version || '',
     };
 
+    if (targetFrameworks.length) {
+      dependency.targetFramework = targetFrameworks;
```

Comment:
We should probably align the naming here, so we would have `dependency.targetFrameworks`.